{"bug_class": "PROVER", "kind": "ERROR", "bug_type": "NULL_DEREFERENCE", "qualifier": "object `null` could be null and is dereferenced by call to `SuperCSVException(...)` at line 123.", "severity": "HIGH", "visibility": "user", "line": 123, "column": -1, "procedure": "void SuperCSVExceptionTest.testConstuctor6()", "procedure_id": "org.supercsv.exception.SuperCSVExceptionTest.testConstuctor6():void.8dd7054cda5273d513adca82b8a8c2c6", "procedure_start_line": 114, "file": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "bug_trace": [{"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 114, "column_number": -1, "description": "start of procedure testConstuctor6()"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 115, "column_number": -1, "description": ""}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 105, "column_number": -1, "description": "start of procedure SuperCSVException(...)"}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 107, "column_number": -1, "description": "Skipping RuntimeException(...): unknown method"}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 108, "column_number": -1, "description": ""}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 109, "column_number": -1, "description": ""}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 110, "column_number": -1, "description": "return from a call to SuperCSVException.<init>(String,CSVContext,CellProcessor,Throwable)"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 116, "column_number": -1, "description": "Skipping assertEquals(...): unknown method"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 117, "column_number": -1, "description": ""}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 118, "column_number": -1, "description": "start of procedure getCsvContext()"}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 119, "column_number": -1, "description": "return from a call to CSVContext SuperCSVException.getCsvContext()"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 117, "column_number": -1, "description": "Skipping assertEquals(...): unknown method"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 118, "column_number": -1, "description": ""}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 137, "column_number": -1, "description": "start of procedure getOffendingProcessor()"}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 138, "column_number": -1, "description": "return from a call to CellProcessor SuperCSVException.getOffendingProcessor()"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 118, "column_number": -1, "description": "Skipping assertEquals(...): unknown method"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 119, "column_number": -1, "description": "Skipping assertEquals(...): unknown method"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 120, "column_number": -1, "description": "Skipping printStackTrace(): unknown method"}, {"level": 0, "filename": "src/test/java/org/supercsv/exception/SuperCSVExceptionTest.java", "line_number": 123, "column_number": -1, "description": ""}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 105, "column_number": -1, "description": "start of procedure SuperCSVException(...)"}, {"level": 1, "filename": "src/main/java/org/supercsv/exception/SuperCSVException.java", "line_number": 107, "column_number": -1, "description": "Skipping getMessage(): unknown method"}], "key": "SuperCSVExceptionTest.java|testConstuctor6|NULL_DEREFERENCE", "node_key": "261b80bba4a54d0242544f042ee55daa", "hash": "3464ce921547b782406c10d25e162b47", "bug_type_hum": "Null Dereference", "censored_reason": ""}